core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/add-repositories.x86_64.sh

```diff
@@ -1,4 +1,15 @@
 #!/usr/bin/env bash -eu
 
+CENTOS_EXTRAS=$(cat <<'EOF'
```

Comment:
Do we need to add this repo?

Reply:
Without this repo there is a following error - https://stackoverflow.com/questions/65878769/cannot-install-docker-in-a-rhel-server